File: frontend/src/types/alcohol.ts
export interface AlcoholEntry {
  id: string
  doses: number
  notes: string | null
  consumed_at: string
  logged_at: string
}

export interface AlcoholDaySummary {
  date: string // "yyyy-MM-dd"
  total_doses: number
  entries: AlcoholEntry[]
}

File: backend/app/models/alcohol.py
from pydantic import BaseModel, Field
from datetime import datetime


class AlcoholEntryCreate(BaseModel):
    doses: int = Field(ge=1)
    notes: str | None = None
    date: str | None = None  # yyyy-MM-dd; default = hoje em Brasília


class AlcoholEntryOut(BaseModel):
    id: str
    doses: int
    notes: str | None
    consumed_at: datetime
    logged_at: datetime


class AlcoholDaySummary(BaseModel):
    date: str  # "yyyy-MM-dd"
    total_doses: int
    entries: list[AlcoholEntryOut]
